### PR TITLE
Update "Compiling for Windows" page.

### PR DESCRIPTION
**Pull request by bruvzg**
- Adds notes related to ARM64 compilation (MinGW-LLVM links and commands).
- Updates ANGLE build instructions.
- Updates custom export templates image.

### CODE REVIEW

**Review by skyace65 — CHANGES_REQUESTED** [review]
The image should be a webp not a png. Also the lines should be the same yellow that's used on other pages of the documentation. The ones you made look a bit...shakey, not sure how else to describe it. If you send me the png without lines I can edit the image for you.

Also does this mean we have out of the box ARM support for Windows in 4.3?

**Comment by bruvzg** [comment]
Switched to `webp` with solid lines.

> Also does this mean we have out of the box ARM support for Windows in 4.3?

Yes, for custom builds it was supported for a long time. And official build containers and scripts were recently updated to include it, so next release should have ARM64 Windows editor and templates.

File: contributing/development/compiling/compiling_for_windows.rst [hunk 1]
```diff
@@ -20,9 +20,14 @@ For compiling under Windows, the following is required:
   **Make sure to enable C++ in the list of workflows to install.**
   If you've already installed Visual Studio without C++ support, run the installer
   again; it should present you a **Modify** button.
+  Supports ``x86_64``, ``x86_32`` and ``arm64``.
```
Comment (AThousandShips) [inline contributing/development/compiling/compiling_for_windows.rst:23]:
```suggestion
  Supports ``x86_64``, ``x86_32``, and ``arm64``.
```

File: contributing/development/compiling/compiling_for_windows.rst [hunk 2]
```diff
@@ -20,9 +20,14 @@ For compiling under Windows, the following is required:
...
 - `MinGW-w64 <https://mingw-w64.org/>`_ with GCC can be used as an alternative to
   Visual Studio. Be sure to install/configure it to use the ``posix`` thread model.
   **Important:** When using MinGW to compile the ``master`` branch, you need GCC 9 or later.
+  Supports ``x86_64`` and ``x86_32`` only.
+- `MinGW-LLVM <https://github.com/mstorsjo/llvm-mingw/releases>`_ with clang can be used as
+  an alternative to Visual Studio and MinGW-w64.
+  Supports ``x86_64``, ``x86_32`` and ``arm64``.
```
Comment (AThousandShips) [inline contributing/development/compiling/compiling_for_windows.rst:30]:
```suggestion
  Supports ``x86_64``, ``x86_32``, and ``arm64``.
```

**Review by skyace65 — CHANGES_REQUESTED** [review]
The color for the outlines still needs to be changed. It should be yellow like in this image: 
ExampleColor

hex code is fffb44 (fffb44ff if there's a transparency value)

**Comment by bruvzg** [comment]
> The color for the outlines still needs to be changed. It should be yellow like in this image:

Done. Seems like it a bit of a mess right now, different parts of the docs use different style, probably should be cleaned up:
Screenshot 2024-07-26 at 10 39 00

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
> > The color for the outlines still needs to be changed. It should be yellow like in this image:
> 
> Done. Seems like it a bit of a mess right now, different parts of the docs use different style, probably should be cleaned up: Screenshot 2024-07-26 at 10 39 00

Yeah, the current goal is to get everything standardized as images need to be updated.

**Review by Calinou — APPROVED** [review]
Looks great!

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! 🙏

**Pull request merged**